byron/ledger/executable-spec/src/Ledger/Delegation.hs

```diff
@@ -417,8 +462,8 @@ instance STS ADELEGS where
         case sig of
           [] -> return st
           (x:xs) -> do
-            ds' <- trans @ADELEGS $ TRC (env, st, xs)
-            ds'' <- trans @ADELEG $ TRC (env, ds', x)
+            ds'  <- trans @ADELEG $ TRC (env, st, x)
```

Comment:
The same here: the patterns match is `x:xs`, while in the spec there is `Γ;c`, which pattern matches on the tail rather on the head.